Complex Radio/dist/metadata.json

```diff
@@ -7,12 +7,12 @@
     "description": {
       "en": "Complex Radio provides quality music to its listeners, we also always have applications open for new presenters!"
     },
-    "url": "complexr.pw",
-    "version": "1.0.1",
+    "url": ["complexr.pw", "complexradio.com"],
+    "version": "1.0.3",
     "logo": "https://i.imgur.com/r633Sz0.png",
     "thumbnail": "https://i.imgur.com/1Iey1co.png",
     "color": "#4a9aa8",
     "tags": ["music","radio","gamers","community","complex","complexradio"],
     "category": "music",
-    "regExp": "(([a-z0-9])+[.])?complexr[.]pw"
+    "regExp": "(([a-z0-9])+[.])?complexradio[.]com"

```

Comment:
the regex will override the url probs so I doubt complexr.pw will still show the presence, please check if it still works on it or not, if not add it to the regex.

Reply:
Ok ill have a look

Reply:
If you want to support both, change the RegExp to `(([a-z0-9])+[.])?(complexradio[.]com|complexr[.]pw)` @TylerCousins